srcpkgs/gnome-flashback/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'gnome-flashback'
+pkgname=gnome-flashback
+version=3.30.0
+revision=1
+only_for_archs="i686 x86_64"
```

Comment:
Please do no use these without a very strong reason, considering the gnome stack is used on x86_64-musl no problems i'd wager this is also the case.